views/HtmlStatic.html

```diff
@@ -39,12 +41,37 @@
       majority: ({ item }) => {
         return item.majority;
       },
+      maxVotes: ({ item }) => {
+        const maxResult = Math.max(...item.candidates.map(candidate => { 
+          if (candidate.votes !== undefined) {
+            return candidate.votes;
+          }
+          return 0;
+        }));
+        if (item.threshold > maxResult) {
+          return item.threshold;
+        }
+        return maxResult;
+      },
+      maxErrorMarginValue: ({ item }) => {
+        const maxErrorMarginValue = Math.max(...item.candidates.map(candidate => { 
+          if (candidate.errorMargin && candidate.errorMargin.upper !== undefined) {
+            return candidate.errorMargin.upper;
+          }
+          return 0;
+        }));
+        if (item.threshold > maxErrorMarginValue) {
+          return item.threshold;
+        }
+        return maxErrorMarginValue;
+      },
       titleClassAttribute: ({ toolRuntimeConfig }) => {
         if (toolRuntimeConfig.displayOptions && toolRuntimeConfig.displayOptions.hideTitle) {
           return 'q-election-executive-title--hidden';
         }
         return '';
       },
+      //TODO: do not random sort but return by array index if no values given
       sortedCandidates: ({ candidates }) => {
```

Comment:
Candidates should if available additionally be sorted on the errorMargin property. See -> 

![Bildschirmfoto 2019-03-20 um 17 23 40](https://user-images.githubusercontent.com/1810384/54701891-230ecb00-4b36-11e9-8293-5e85388f7779.png)

Reply:
Thanks for spotting. This is now fixed.